style-dictionary/classic/index.ts

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 import { ThemeBuilder } from '@cloudscape-design/theming-build';
 import { createColorMode, createDensityMode, createMotionMode } from '../utils/modes';
-import { createTopNavigationContext } from '../utils/contexts';
+import { createTopNavigationContext, createFlashbarContext } from '../utils/contexts';
```

Comment:
`createFlashbarContext` added from https://github.com/cloudscape-design/components/pull/266, for internal color update CR-75880450